packages/typescript-estree/src/ast-converter.ts

```diff
@@ -1,10 +1,3 @@
-/**
- * @fileoverview Converts TypeScript AST into ESTree format.
- * @author Nicholas C. Zakas
- * @author James Henry <https://github.com/JamesHenry>
- * @copyright jQuery Foundation and other contributors, https://jquery.org/
- * MIT License
```

Comment:
Should this stay here to acknowledge copyright?

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/LICENSE

Reply:
I would agree if we had them on every file, but it was only on a few files (like 4?).
The point of the comment is usually for in case a single file gets leaked, so the "licence is maintained", but unless we're consistent that feels like it doesn't apply.

Reply:
Still, do we need to cite the original authors of the files when they came from before the current license?

Reply:
IANAL - so I don't know exactly, but my gut is no?
I don't think there was a per-file licence attributed to each file's original author, instead it defaulted to the project's licence, which has been maintained (though it has changed from the jquery foundation to the js foundation)